@xen-orchestra/web-core/lib/components/ui/table-pagination/UiTablePagination.vue

```diff
@@ -0,0 +1,207 @@
+<!-- v2 -->
+<template>
+  <div class="ui-table-pagination">
+    <span class="typo p3-regular label">
+      {{ $t('core.select.n-object-of', { from: startIndex, to: endIndex, total: totalItems }) }}
+    </span>
+
+    <div class="buttons-content">
```

Comment:
I think `container` would be more appropriate

```suggestion
    <div class="buttons-container">
```